packages/v4/src/content/design-guidelines/components/time picker/time-picker.md

```diff
@@ -0,0 +1,104 @@
+---
+id: Time picker
+section: components
+related: ["Calendar month", "Date picker"]
+---
+
+A **time picker** allows users to select a time from a list of options. Time can be displayed in hours and minutes, but not in seconds.
+
+## Elements
+
+<img src="./img/Time picker- elements.png" alt="Time picker elements" width="362"/>
+
+1. **Time field**: Displays a user’s selected time. Always fill the time field with a placeholder time format (HH:MM) or a pre-selected time before users select their time by entering the field, typing ahead, scrolling, and selecting their desired time from the menu. The time field should never be empty.
```

Comment:
Would break this sentence up a little bit and modify: 

"Displays a user's selected time. Always fill the time field with a placeholder time format (HH:MM) or a pre-selected time. Users can choose a time by either typing in a time, or selecting one from the menu list of time options. The time field should never be empty."